README.md

```diff
@@ -1,48 +1,54 @@
 # OpenMS NuXL App [![Open Template!](https://static.streamlit.io/badges/streamlit_badge_black_white.svg)](https://openms-template.streamlit.app/)
-**powered by:**
-
-<img src="assets/OpenMS.png" width=20%>
 
-This is a web application of NuXL search engine build in [OpenMS](https://openms.de/).
+Welcome to the OpenMS NuXL App, a web application for the NuXL protein-nucleic acid search engine built using [OpenMS](https://openms.de/) and [pyOpenMS](https://pyopenms.readthedocs.io/en/latest/).
 
 ## NuXL
 
-- **Description:** NuXL, A dedicated software package for the analysis of XL-MS data obtained from UV and chemically crosslinked protein–RNA/DNA samples. It allows for reliable, FDR-controlled assignment of protein–nucleic acid crosslinking sites from samples treated with UV light or chemical crosslinkers and offers user-friendly matched spectra visualization including ion annotations.[more](https://ssp2022.com/index.php/timetable/event/henning-urlaub/#:~:text=NuXL%20is%20available%20in%20the,spectra%20visualization%20including%20ion%20annotations.)
+- **Description:** NuXL is a dedicated software package designed for the analysis of XL-MS (cross-linking mass spectrometry) data obtained from UV and chemically crosslinked protein–RNA/DNA samples. This powerful tool allows for reliable, FDR-controlled assignment of protein–nucleic acid crosslinking sites in samples treated with UV light or chemical crosslinkers. It offers user-friendly matched spectra visualization, including ion annotations.[more](https://ssp2022.com/index.php/timetable/event/henning-urlaub/#:~:text=NuXL%20is%20available%20in%20the,spectra%20visualization%20including%20ion%20annotations.)
   
+**powered by:**
+
+<img src="assets/OpenMS.png" width=20%>
   
-## Installation
+## Running NuXL locally: Installation as stand-alone tool
 ### Windows
-1. Download and extract the [OpenMS-app.zip](https://github.com/Arslan-Siraj/nuxl-app/actions) file from latest successfull action.
+1. To get started, download and extract the [OpenMS-app.zip](https://github.com/Arslan-Siraj/nuxl-app/actions) file from latest successfull action.
 2. Run the `run_app.exe`
 3. Use app in your default browser
    
 ## Quickstart 
 
-### Workspaces
+You can start right away analyzing your data by following the steps below:
+
+### 1. Create a workspace
 On the left side of this page you can define a workspace where all your data including uploaded `mzML` files will be stored. Entering a workspace will switch to an existing one or create a new one if it does not exist yet. In the web app, you can share your results via the unique workspace ID. Be careful with sensitive data, anyone with access to this ID can view your data.
```

Comment:
is this correct?  I think users will not understand the difference local vs online